plugins/modules/nmcli.py

```diff
@@ -293,6 +293,13 @@
         type: str
         choices: [ignore, auto, dhcp, link-local, manual, shared, disabled]
         version_added: 2.2.0
+    may_fail6:
+        description:
+            - If you need I(ip6) configured before C(network-online.target) is reached, set this option to C(false).
+            - This option applies when C(method6) is not C(disabled) or C(ignore).
+        type: bool
+        default: true
```

Comment:
Usually new options should not have a default, to avoid problems with idempotency. Can you remove this and only make the module change the value if this option is specified?

Reply:
Agree. Done.